music21/converter/subConverters.py

```diff
@@ -806,19 +804,22 @@ def runThroughMusescore(self, fp, subformats=None, **keywords):
         fpOut = fp[0:len(fp) - 3]
         fpOut += subformatExtension
 
-        musescoreRun = '"' + musescorePath + '" ' + fp + " -o " + fpOut + " -T 0 "
+        musescoreRun = '' + musescorePath + ' ' + fp + " -o " + fpOut + " -T 0 "
         if 'dpi' in keywords:
             musescoreRun += " -r " + str(keywords['dpi'])
 
         if common.runningUnderIPython():
             musescoreRun += " -r " + str(defaults.ipythonImageDpi)
 
-        storedStrErr = sys.stderr
-        fileLikeOpen = six.StringIO()
-        sys.stderr = fileLikeOpen
-        os.system(musescoreRun)
-        fileLikeOpen.close()
-        sys.stderr = storedStrErr
+        import subprocess
+        command = musescoreRun.split()
+        try:
+            env = dict(os.environ, QT_QPA_PLATFORM='offscreen')
+            output = subprocess.check_output(command
+                             , stderr=subprocess.STDOUT, env=env)
+        except subprocess.CalledProcessError as cpe:
+            print("program error:", cpe, file=sys.stderr)
```

Comment:
use environLocal.warn() instead of print -- this lets people capture all warnings directly along with where they come from (they may have redirected it to somewhere instead of sys.stderr, for instance when running music21 as a service).  Or don't print anything simply let the message bubble up.